src/main/index.ts

```diff
@@ -50,7 +51,7 @@ export async function main(): Promise<void> {
             responseHeaders: Object.assign(
               {
                 ...details.responseHeaders,
-                "Content-Security-Policy": ["img-src *"],
+                "Content-Security-Policy": [`img-src ${getImgSrcCsp()}`],
```

Comment:
Can we add a comment here to also update `devContentSecurityPolicy` in `forge.config.js` too? I'm concerned they'll get out of sync because it's easy to forget about updating it there.